hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/SnapshotDiffJob.java

```diff
@@ -187,6 +215,13 @@ public String toString() {
     if (StringUtils.isNotEmpty(reason)) {
       sb.append(", reason: ").append(reason);
     }
+    if (status.equals(JobStatus.IN_PROGRESS) && subStatus != null) {
+      sb.append(", subStatus: ").append(status);
+      if (subStatus.equals(SubStatus.OBJECT_ID_MAP_GEN_FSO) ||
+          subStatus.equals(SubStatus.OBJECT_ID_MAP_GEN_OBS)) {
+        sb.append(", keysProcessedPercent: ").append(keysProcessedPct);
```

Comment:
```suggestion
        sb.append(String.format(", keysProcessedPercent: %.2f", keysProcessedPct));
```
Let's display the percent upto 2 decimal places.

Reply:
done.